<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OwnCloud - Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="group_account">
     <property name="title">
      <string>Account</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_urlFormat">
        <property name="text">
         <string>Server URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_url"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_usernameFormat">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_username"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_passwordFormat">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="input_password">
        <property name="echoMode">
         <enum>PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_connection">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="button_authenticate">
        <property name="text">
         <string>Test Connection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_status" native="true">
        <layout class="QFormLayout" name="form_status">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_credentials">
           <property name="text">
            <string>Credentials:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_status">
           <property name="text">
            <string>Unknown</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_name">
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_remotePath">
        <property name="text">
         <string>Upload Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_path"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_nameFormat">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_name"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_result">
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_example">
        <property name="text">
         <string>example</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_clipboard">
     <property name="title">
      <string>Clipboard</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radio_owncloud">
        <property name="text">
         <string>Copy OwnCloud link</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_directlink">
        <property name="text">
         <string>Copy direct link</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_dontcopy">
        <property name="text">
         <string>Don't copy link</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_authenticate</tabstop>
  <tabstop>input_url</tabstop>
  <tabstop>input_username</tabstop>
  <tabstop>input_password</tabstop>
  <tabstop>radio_owncloud</tabstop>
  <tabstop>radio_directlink</tabstop>
  <tabstop>radio_dontcopy</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
